modulus/datapipes/climate/climate_hdf5.py

```diff
@@ -0,0 +1,590 @@
+# Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
No reason for the naming really and would be open to something else. I think we should keep the current ERA5 dataloader though until at least after we make the unified recipes. This can just be a smallish PR working towards that.